main/createUiDefinition.json

```diff
@@ -92,7 +92,7 @@
               "virtualNetwork": "Only virtual networks in the currently selected subscription and location are listed. If you choose to create a new virtual network, it will be created in the same subscription, location, and resource group as the storage account."
             },
             "defaultValue": {
-              "addressPrefixSize": "/8"
+              "addressPrefixSize": "/16"
```

Comment:
```suggestion
              "addressPrefix": "/16"
```

Reply:
Removed default value as not needed more.